docs/modelserving/v1beta1/transformer/collocation/README.md

```diff
@@ -143,3 +152,240 @@ curl -v -H "Host: ${SERVICE_HOSTNAME}" -H "Content-Type: application/json" -d $I
     * Connection #0 to host localhost left intact
     {"predictions":[2]}
     ```
+
+## Collocation with Runtime
+### Deploy the InferenceService
+
+Since, the predictor and the transformer are in the same pod, they need to listen on different ports to avoid conflict. `Transformer` is configured to listen on port 8080 (REST) and 8081 (GRPC) 
+while, `Predictor` listens on port 8085 (REST). `Transformer` calls `Predictor` on port 8085 via local socket. 
+Deploy the `Inferenceservice` using the following command:
+
+Note that, readiness probe is specified in the transformer container. This due to the limitation of Knative. You can provide `--enable_predictor_health_check` argument to allow the transformer container to check the predictor health as well. This will make sure that both the containers are healthy before the isvc is marked as ready.
```

Comment:
```suggestion
Note that, HTTP readiness probe is specified in the transformer container to override the default TCP readiness probe. You can provide `--enable_predictor_health_check` argument to allow the transformer container to check the predictor health as well. This will make sure that both the containers are healthy before the isvc is marked as ready.
```